binding-generator/src/generator.rs

```diff
@@ -433,6 +434,11 @@ impl Generator {
 		args.push("-includeocvrs_common.hpp".into());
 		// need to have c++14 here because VS headers contain features that require it
 		args.push("-std=c++14".into());
+		// allow us to use some custom clang args
+		let clang_args = env::var_os("OPENCV_CLANG_ARGS").unwrap_or_default();
+		clang_args.to_string_lossy().split_whitespace().map(String::from).for_each(|i| {
```

Comment:
Thanks for the PR! Regarding the splitting the arguments, can you please try employing `Shlex` as it's now done in [`cmake_probe.rs`](https://github.com/twistedfall/opencv-rust/blob/master/build/cmake_probe.rs#L125)? Otherwise this naive `split_whitespace` will be handling for example "quoted spaced values" incorrectly.